app/assets/javascripts/discourse/tests/acceptance/sidebar-test.js

```diff
@@ -5,6 +5,7 @@ import { click, currentRouteName, visit } from "@ember/test-helpers";
 import {
   acceptance,
   exists,
+  publishToMessageBus,
```

Comment:
I think it is some work in progress leftover

Reply:
oops :+1: